robottelo/config/base.py

```diff
@@ -498,6 +496,8 @@ def read(self, reader):
 
     def validate(self):
         """Validate AzureRM settings."""
+        from robottelo.constants import AZURERM_VALID_REGIONS
```

Comment:
I believe this and `VALID_GCE_ZONES` can be brought back to top of file, since circular dependency import is no longer a concern?

Reply:
Done!